### ISSUE TITLE
compilerOptions.typeRoots in tsconfig.json is missing from the options resolver.

### ISSUE THREAD

**Issue by mientjan** [TypeStrong/grunt-ts#383]
When your tsconfig and your node_modules have a completely different path from each other you need to override the typeRoots in your tsconfig. This option is not available in grunt-ts not even in the latest version. 6.0.0-beta.3

```
{
  "compilerOptions": {
    "typeRoots":[
      "../../../../../../tools/build/node_modules/@types/"
    ]
  }
}
```

**Comment by ksjogo** [comment]
You can add them as additional param for the moment:
```
		ts: {
			default : {
				tsconfig: true,
				options: {
            verbose: true,
            additionalFlags: '--typeRoots "node_modules/@types,types"'
				}
			}
		},

```

**Comment by alonn24** [comment]
+1

**Comment by nycdotnet** [comment]
It's not on npm yet but the WIP PR has it and should be out this week.  Until then, additionalFlags is the way to go.

**Comment by nycdotnet** [comment]
This is implemented in 6.0.0-beta.6 on npm.  Please give it a whirl and let me know if you hit any issues.

**Comment by nycdotnet** [comment]
I am going to close this issue as I believe it is resolved in the latest beta.  Please feel free to reopen/comment.  Thank you!
